ecbuild failure to detect netcdf-fortran when building standalone.

Ecbuild fails when building standalone:

CMake Error at /home/h01/frwd/cylc-run/mi-be984/share/installs/ecbuild/share/ecbuild/cmake/ecbuild_add_library.cmake:308 (add_library):
  Target "monio" links to target "NetCDF::NetCDF_Fortran" but the target was
  not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
Call Stack (most recent call first):
  /home/h01/frwd/cylc-run/mi-be984/share/installs/ecbuild/share/ecbuild/cmake/ecbuild_add_library.cmake:667 (ecbuild_add_library_impl)
  src/CMakeLists.txt:59 (ecbuild_add_library)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,7 +30,7 @@ set(MONIO_LINKER_LANGUAGE CXX)
 find_package(jedicmake QUIET)  # Prefer find modules from jedi-cmake
 find_package(MPI REQUIRED COMPONENTS CXX)
 find_package(HDF5 REQUIRED COMPONENTS)
-find_package(NetCDF COMPONENTS CXX)
+find_package(NetCDF COMPONENTS Fortran CXX)
 find_package(eckit 1.16.1 REQUIRED COMPONENTS MPI)
 find_package(atlas 0.20.2 REQUIRED)
 find_package(oops 1.0.0 REQUIRED)

